### PR TITLE
Bug 1872972: Make efficiency cards consistent

### PR DESCRIPTION
**Pull request by a2batic**
Screenshot from 2020-09-14 14-41-50

Signed-off-by: Kanika Murarka <kmurarka@redhat.com>

### CODE REVIEW

**Comment by afreen23** [comment]
Please attach screenshots for the change.
Thanks!

**Comment by a2batic** [comment]
@afreen23 updated the description, please review.

**Review by afreen23 — CHANGES_REQUESTED** [review]
We need to rename `efficiency` variables to `compressionRatio` , alongwith changing filename and changes in `queries.ts` file of noobaa plugin.

**Comment by a2batic** [comment]
@afreen23 addressed review comments, please review.

**Comment by afreen23** [comment]
Nit: As per convention , we name the files and folders of cards by the name of card. So `object-storage-effiiciency-card` would be `storage-efficiency-card`

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/storage-efficiency-card/storage-efficiency-card-item.tsx [hunk 1]
```diff
@@ -2,11 +2,11 @@ import * as React from 'react';
 import { FieldLevelHelp } from '@console/internal/components/utils';
 
 export const EfficiencyItemBody: React.FC<EfficiencyItemBodyProps> = React.memo(
-  ({ title, infoText, isLoading, error, getStats }) => {
+  ({ stats, title, infoText, isLoading, error, getStats }) => {
     let status: React.ReactElement;
     if (isLoading) {
       status = <div className="skeleton-text ceph-storage-efficiency-card__item-body--loading" />;
-    } else if (error) {
+    } else if (error || stats === 0) {
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/storage-efficiency-card/storage-efficiency-card-item.tsx:9]: Noobaa compresion ratio can be negative certain times
```suggestion
    } else if (error || stats <= 0) {
```

**Review by afreen23 — CHANGES_REQUESTED** [review]
We had this check for data reduction card previously, we need to consider this

**Review by afreen23 — APPROVED** [review]
/lgtm

**Pull request merged**